### ISSUE TITLE
"Reissued any over long jobs"  message is alarming

### ISSUE THREAD

**Issue by diekhans** [DataBiosphere/toil#3014]
Toil printing this message even when nothing is reissued leads people to think their workflow is stuck

```
Reissued any over long jobs
Reissued any over long jobs
Reissued any over long jobs

```

┆Issue is synchronized with this [Jira Task](https://ucsc-cgl.atlassian.net/browse/TOIL-529)
┆Issue Number: TOIL-529

**Comment by DailyDreaming** [comment]
Maybe something simple like: `# jobs still running.  # jobs queued.`

**Comment by adamnovak** [comment]
This was fixed by #3044.
